src/importexport/mei/internal/meiexporter.cpp

```diff
@@ -893,6 +893,17 @@ bool MeiExporter::writeLayer(track_idx_t track, const Staff* staff, const Measur
     meiLayer.SetN(static_cast<int>(track2voice(track) + 1));
     meiLayer.Write(m_currentNode, this->getLayerXmlId());
 
+    const Measure* mmR1 = measure->coveringMMRestOrThis();
+    if (mmR1->mmRestFirst()) {
+        pugi::xml_node multiRestNode = m_currentNode.append_child();
+        libmei::MultiRest meiMultiRest;
+        meiMultiRest.SetNum(mmR1->mmRestCount());
+        std::string xmlId = this->getXmlIdFor(mmR1, 'm');
+        meiMultiRest.Write(multiRestNode, xmlId);
```

Comment:
To be more inline with the rest of the exporter structure, make a `writeMultiRest` method for this.